### PR TITLE
Animation Cleanup: En_F*

### PR DESCRIPTION
**Pull request by engineer124**
I also included `z_en_minifrog` in here due to its parallels to `z_en_fg`

### CODE REVIEW

File: src/overlays/actors/ovl_En_Minifrog/z_en_minifrog.h [hunk 1]
```diff
@@ -2,33 +2,34 @@
...
-    /* 0x03 */ MINIFROG_BLUE,           // Southern Swamp
-    /* 0x04 */ MINIFROG_WHITE,          // Laundry Pool
-    /* 0xFF */ MINIFROG_INVALID = 0xFF // -1
-} MinifrogType;
+    /* 0x00 */ FROG_YELLOW,         // Mountain Village
+    /* 0x01 */ FROG_CYAN,           // Woodfall Temple
+    /* 0x02 */ FROG_PINK,           // Great Bay Temple
+    /* 0x03 */ FROG_BLUE,           // Southern Swamp
+    /* 0x04 */ FROG_WHITE,          // Laundry Pool
+    /* 0xFF */ FROG_NONE = 0xFF // -1
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Minifrog/z_en_minifrog.h:19]: Probably don't need as much whitespace now.

**Review by hensldm — APPROVED** [review]
Just 1 small nit

File: src/overlays/actors/ovl_En_Fu/z_en_fu.c [hunk 2]
```diff
@@ -73,14 +72,26 @@ static Vec3f D_80964B0C = { 0.0f, 60.0f, -8.0f };
...
+    /* 7 */ HONEY_DARLING_ANIM_MAX
+} HoneyAndDarlingAnimation;
+
+static AnimationInfo sAnimationInfo[HONEY_DARLING_ANIM_MAX] = {
+    { &gHoneyAndDarlingIdleAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },          // HONEY_DARLING_ANIM_IDLE
+    { &gHoneyAndDarlingCupCheeksLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f }, // HONEY_DARLING_ANIM_CUP_CHEEKS
+    { &gHoneyAndDarlingHugLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },       // HONEY_DARLING_ANIM_HUG
+    { &gHoneyAndDarlingGameDanceLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, 0.0f },  // HONEY_DARLING_ANIM_GAME_DANCE
+    // HONEY_DARLING_ANIM_HOLD_HANDS_MORPH
     { &gHoneyAndDarlingHoldHandsLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Fu/z_en_fu.c:92]: The enum comment on a different line is a bit annoying. What about using this comment style for all the entries of this array instead?

**Review by isghj5 — APPROVED** [review]
My one complaint can wait for a future documentation pass

File: assets/xml/objects/object_fish.xml [hunk 3]
```diff
@@ -20,22 +20,22 @@
...
+        <Limb Name="object_fish_Standardlimb_002930" Type="Standard" EnumName="FISHING_FISH_LIMB_09" Offset="0x2930" />
+        <Limb Name="object_fish_Standardlimb_00293C" Type="Standard" EnumName="FISHING_FISH_LIMB_0A" Offset="0x293C" />
+        <Limb Name="object_fish_Standardlimb_002948" Type="Standard" EnumName="FISHING_FISH_LIMB_0B" Offset="0x2948" />
+        <Limb Name="object_fish_Standardlimb_002954" Type="Standard" EnumName="FISHING_FISH_LIMB_0C" Offset="0x2954" />
+        <Limb Name="object_fish_Standardlimb_002960" Type="Standard" EnumName="FISHING_FISH_LIMB_0D" Offset="0x2960" />
+        <Limb Name="object_fish_Standardlimb_00296C" Type="Standard" EnumName="FISHING_FISH_LIMB_0E" Offset="0x296C" />
+        <Limb Name="object_fish_Standardlimb_002978" Type="Standard" EnumName="FISHING_FISH_LIMB_0F" Offset="0x2978" />
+        <Skeleton Name="gFishingFishSkel" Type="Flex" LimbType="Standard" LimbNone="FISHING_FISH_LIMB_NONE" LimbMax="FISHING_FISH_LIMB_MAX" EnumName="FishingFishLimb" Offset="0x29C0" />
         <Texture Name="object_fish_Tex_0029D0" OutName="tex_0029D0" Format="rgba16" Width="8" Height="16" Offset="0x29D0" />
```
[the remark this answers is not in the document]
Comment (engineer124) [reply assets/xml/objects/object_fish.xml:39]: Yeah, I see what you mean. In this case, `gFishingFishSkel`, `gFishingLoachSkel`, and `gFishingOwnerSkel` were already named and I was just transferring the names to the limbs of the skeleton. I could see `gFishingFishSkel` being named to the bass/fry in later passes if that makes sense.

File: src/overlays/actors/ovl_En_Fu/z_en_fu.c (hunk 2 above)
Comment (engineer124) [reply src/overlays/actors/ovl_En_Fu/z_en_fu.c]: Sure. Some of these animations once named can get quite long, so I could see this being a recurring issue, just something to think about

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**